.github/workflows/build.yml

```
@@ -0,0 +1,36 @@
+on: push
```

For this to work correctly on this repo, where most PRs come from forks, you need to do `on: pull_request`.

```suggestion
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```